environments/gcp/flyte-core/README.md

```diff
@@ -88,11 +88,22 @@ flyte-core        <none>   flyteontf.uniondemo.run   35.237.42.230   80, 443   3
 flyte-core-grpc   <none>   flyteontf.uniondemo.run   35.237.42.230   80, 443   3m1s
 ```
 3. Create a DNS `A` record in a zone you own, pointing to the Ingress IP.
+
+>NOTE: it may take a while before `cert-manager` can issue a certificate for your deployment, especially because for that process to work,
+the FQDN of your deployment needs to be resolvable and DNS propagation takes time. 
+
 4. Update your `$HOME\.flyte\config,yaml` and make `endpoint` your DNS name:
 ```yaml
 ...
 #Example
 endpoint: dns:///flyteontf.uniondemo.run 
+insecure: false #it means, the connection uses SSL, even if it's a temporary cert-manager cert.
+
+#Uncomment only if you want to test CLI commands and the certificate is not generated yet.
+# You can confirm the cert by either going to the UI (a valid certificate should be used) or
+#from your terminal: kubectl get challenges.acme.cert-manager.io -n flyte (there should not be any pending challenge)
+
+#insecureSkipVerify: true 
```

Comment:
Minor:
We could explain here, that with this option, SSL is used but the certificate is not checked. Am I mistaken that with `insecureSkipVerify: false` we would also have to provide a path to the respective `cert.pem`?

Reply:
Good question, I haven't been required to provide the cert to use it. Nevertheless, it seems to be the case (according to https://github.com/flyteorg/flytekit/blob/5c6802c97388c40d0431e4fc308e2ed651528952/flytekit/clients/auth_helper.py#L159C8-L159C8)